### PR TITLE
Create Plots.jl_Examples.md

### PR DESCRIPTION
**Pull request by NandVinchhi**
These are the Basic examples from Plots.jl converted to Makie.jl for the purpose of uniformity and convenience.
Created by Nand Vinchhi, with reference to ComputerM's pull request for the purpose of Google Code In.

### CODE REVIEW

File: docs/src/basic-tutorials.md [hunk 1]
```diff
@@ -43,14 +43,23 @@ It is worth noting initally that if you run a Makie.jl example and nothing shows
...
+plotting random dots on a graph using __scatter__.
+
 @example_database("Tutorial simple scatter")
 
 @example_database("Tutorial markersize")
 
+@example_database("Tutorial advanced scatter plot")
+
 ### Line plot
+Line plot can be used do plot any line. Attributes can be used to control width, color etc. of the line.
```
Comment (logankilpatrick) [inline docs/src/basic-tutorials.md]:
```suggestion
A line plot can be used to plot any line. Attributes can be used to control width, color etc. of the line. 
```

**Comment by SimonDanisch** [comment]
Very nice! :)

**Comment by SimonDanisch** [comment]
Hey I updated the snake example... You should really try not to make new plots in a loop like that ;) 
I hope I didn't break anything - I couldn't really compare, because it wasn't running for me... There is a bug, where the snake just starts moving as a whole, instead of following the path...

**Comment by NandVinchhi** [comment]
I have made the whole code smoother and less laggy. Will look into the snake head bug.

**Comment by NandVinchhi** [comment]
Thnx for the feedback btw!

**Comment by SimonDanisch** [comment]
Why did you revert all my fixes?!

**Pull request closed without merging**